bin/plugin-helpers.js

```diff
@@ -15,33 +23,53 @@ enableCollectingUnknownOptions(
     .command('start')
     .description('Start kibana and have it include this plugin')
     .on('--help', docs('start'))
-    .action(run('start'))
+    .action(taskRunner(function (command) {
+      run('start', {
+        flags: command.unknownOptions
+      });
+    }))
 );
 
 program
   .command('build')
   .description('Build a distributable archive')
   .on('--help', docs('build'))
-  .action(run('build'));
+  .action(taskRunner(function () {
+    run('build');
+  }));
 
 program
-  .command('test')
+  .command('test [files...]')
```

Comment:
This doesn't feel right. Lets just keep `test` a shortcut for "all test files, browser and server"